Sources_v3/StreamChatUI/ChatChannelList/ChatReadStatusCheckmarkView.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+open class ReadStatusCheckmarkView: UIImageView {
+    
+    public enum Status {
+        case read, unread, empty
+    }
+    
+    public var status: Status = .empty {
+        didSet {
+            updateContent()
+        }
+    }
+    
+    public var readTintColor: UIColor { tintColor }
+    
+    public var unreadTintColor: UIColor = .gray
```

Comment:
Shouldn't we react on color changes and call `updateContent` to have the UI up-to-date?